schema/deploy/computed_columns/project_manager_names_concatenated.sql

```diff
@@ -0,0 +1,31 @@
+-- Deploy cif:computed_columns/project_manager_names_concatenated to pg
+
+-- instead returns text, concatenated, it'll be an empty string if there are no managers. Then it wouldn't need to take an argument, filter for empty string and not null (check filters for null or empty options). includes insensitive empty string (potentially consider refactoring to use tvector later)
+
+begin;
+
+create or replace function cif.project_manager_names_concatenated(cif.project)
+returns text
+as $$
+declare
+  index int;
+  names text := '';
+begin
+  for index in 1..4
+  loop
+    select concat(names, (select cif.cif_user_full_name(
```

Comment:
I think using `string_agg` instead of concat here will simplify things.
https://www.postgresqltutorial.com/postgresql-aggregate-functions/postgresql-string_agg-function/